hathorlib/token_creation_tx.py

```diff
@@ -174,6 +175,34 @@ def verify_token_info(self) -> None:
         if clean_token_string(self.token_symbol) == clean_token_string(settings.HATHOR_TOKEN_SYMBOL):
             raise TransactionDataError('Invalid token symbol ({})'.format(self.token_symbol))
 
+    def is_nft_creation(self) -> bool:
+        """Returns True if it's an NFT creation transaction"""
+        # We will check the outputs to validate that we have an NFT standard creation
+        # https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+        if len(self.outputs) < 2:
+            # NFT creation must have at least a DataScript output (the first one) and a Token P2PKH output
+            return False
+
+        first_output = self.outputs[0]
+        parsed_first_output = DataScript.parse_script(first_output.script)
+
+        if parsed_first_output is None:
+            # First output is not a DataScript output
+            return False
+
+        if first_output.value != 1 or first_output.token_data != 0:
+            # NFT creation DataScript output must have value 1 and must be of HTR
+            return False
+
+        for output in self.outputs[1:]:
+            parsed_output = P2PKH.parse_script(output.script)
+            if parsed_output is None or ((output.token_data & TxOutput.TOKEN_INDEX_MASK) not in [0, 1]):
```

Comment:
We should replace `output.token_data & TxOutput.TOKEN_INDEX_MASK` by `output.get_token_index()`.

Reply:
Changed https://github.com/HathorNetwork/python-hathorlib/pull/6/commits/0bb33939e9ec5733116de029591e12c959f4aebc

Reply:
Great! Thanks!